cpp/src/arrow/compute/kernels/vector_sort.cc

```diff
@@ -88,6 +88,28 @@ struct PartitionIndices {
   }
 };
 
+template <typename ArrayType, typename VisitorNotNull, typename VisitorNull>
+inline void VisitRawValueInline(const ArrayType& values,
```

Comment:
"VisitRawValuesInline" (plural) would be better

Reply:
done